GirafRest/Controllers/AccountController.cs

```diff
@@ -167,11 +167,18 @@ public async Task<ActionResult> Register([FromBody] RegisterDTO model)
             {
                 if (department != null)
                 {
-                    if (model.Role == GirafRoles.Citizen)
-                        AddGuardiansToCitizens(user);
-                    else if (model.Role == GirafRoles.Guardian)
-                        AddCitizensToGuardian(user);
-                    // save changes
+                    switch (model.Role)
+                    {
+                        case GirafRoles.Citizen:
+                            AddGuardiansToUser(user);
+                            break;
+                        case GirafRoles.Guardian:
+                            AddCitizensToGuardian(user);
+                            break;
+                        case GirafRoles.Trustee:
```

Comment:
if the content of this case is the same as another case, could it not just be a fall-through with that case

Reply:
Det kunne man godt med `case GirafRoles.Citizen` og `case GirafRoles.Trustee` lige nu. Men jeg tror også det vil gøre kontrolstrukturen mere utydelig + det vil skulle bruges i et fremtidigt issue, hvor logikken for trustee skal ændres.

Hvis i gerne vil havde det ændret, så ændre jeg det.

_Forstår jeg spørgsmålet korrekt?_